src/FlightMap/Widgets/PhotoVideoControl.qml

```diff
@@ -34,7 +34,7 @@ Rectangle {
 
     // The following properties relate to a simple camera
     property var    _flyViewSettings:                           QGroundControl.settingsManager.flyViewSettings
-    property bool   _simpleCameraAvailable:                     !_mavlinkCamera && _activeVehicle && _flyViewSettings.showSimpleCameraControl.rawValue
+    property bool   _simpleCameraAvailable:                     !_mavlinkCamera && _activeVehicle && _flyViewSettings.showSimpleCameraControl.rawValue && _videoStreamIsStreaming
```

Comment:
This is not a correct fix. This will likely break taking photos when you are in simple camera mode. This change would only allow you to take a photo when video is streaming which doesn't make sense. A simple cameras can do both photos and video.

Reply:
Ah my mistake. Did not realize QGroundControl supported taking photos with a simple camera when it is not actively streaming.

Reply:
Undone